### ISSUE TITLE
Add option to install from file:/// URLs

### ISSUE THREAD

**Issue by ivotron** [frictionlessdata/dpm#55]
Add an option that allows to reference a `datapackage.json` file stored locally

**Comment by rgrp** [comment]
Good idea - a bit of a pain because request does not support file:/// IIRC ...
